tests/20-main.spec.js

```diff
@@ -261,7 +261,7 @@ describe('statusTypeMatches', () => {
     err.message.should.contain('"credentialStatus" is invalid');
   });
 
-  it('should not match when "CONTEXTS.VC_BRL_V1" is not in ' +
+  it('should not match when "CONTEXTS.VC_BSL_V1" is not in ' +
```

Comment:
This is true for 1.x VCs ... if we don't have a test for 2.x VCs, we should add one such that if the VC 2.0 context is present, then the BSL context needn't be (in order for the status to match).

Reply:
I have reworded this test to include "or VC 2". We will need to add new tests for all of the VC 2 work but this rounds out he VC 1 work I believe